.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
I don't think a feature request has to necessarily be related to a problem, it could just be an enhancement. This template could be reworded some to make a feature seem less like it is fixing a problem.